src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -18,19 +18,33 @@
 using Microsoft.ML.Runtime.Model.Onnx;
 using Microsoft.ML.Runtime.Model.Pfa;
 using Newtonsoft.Json.Linq;
+using Microsoft.ML.Core.Data;
```

Comment:
mildly shock

Reply:
Indeed Pete, you must learn to sort. :D :D :D

---
In reply to: [217230842](https://github.com/dotnet/machinelearning/pull/896#discussion_r217230842) [](ancestors = 217230842)